### PR TITLE
feat(i18n): add norwegian stuff

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

### CODE REVIEW

File: src/languages/index.ts [hunk 1]
```diff
@@ -3,14 +3,16 @@ import { ExtendedHandler as DeDeHandler } from './de-DE/constants';
...
 import { ExtendedHandler as NlNlHandler } from './nl-NL/constants';
 
 export const handlers = new Map<string, Handler>([
 	['de-DE', new DeDeHandler()],
 	['en-US', new EnUsHandler()],
 	['en-GB', new EnGbHandler()],
 	['es-ES', new EsEsHandler()],
-	['nl-NL', new NlNlHandler()]
+	['nl-NL', new NlNlHandler()],
+	['nb-NO', new NbNoHandler]
```
Comment (kyranet) [inline src/languages/index.ts]:
```suggestion
	['nb-NO', new NbNoHandler()],
	['nl-NL', new NlNlHandler()]
```

Comment (ghost) [reply src/languages/index.ts]: oh right

**Review by kyranet — APPROVED** [review]
Ordinal pattern is the following:

```javascript
1     -ste
2     -ndre
3     -dje
4-5   -de
6     -te
7-10  -de
11-12 -te
13-20 -de
21    -ste
22    -ndre
... // Repeat 1-10 cycle
```

The code checks out with the pattern, and the configuration of the handler is complete and correct, LGTM, and congratulations for your first contribution! ❤️🎉

**Comment by ghost** [comment]
It's not actually my first contribution, I did one yesterday, but thank you!

**Pull request merged**